railties/lib/rails/code_statistics.rb

```diff
@@ -4,6 +4,31 @@
 require "active_support/core_ext/enumerable"
 
 class CodeStatistics # :nodoc:
+  DIRECTORIES = [
```

Comment:
This is technically private, so we're deprecating (also a private) constant with another 🤔

Reply:
Yes. `::CodeStatistics::TEST_TYPES ` is used similarly:
https://github.com/rspec/rspec-rails/blob/3f4b1e03e6331c78780081a65deb4055692511b1/lib/rspec/rails/tasks/rspec.rake#L43-L44
Maybe a setting would be better? Not sure I can get that working with the DeprecationProxy.

Reply:
I'm using a setting now instead.